pkg/scheduler/plugins/loadaware/load_aware.go

```diff
@@ -170,7 +175,7 @@ func (p *Plugin) Filter(ctx context.Context, state *framework.CycleState, pod *c
 	return nil
 }
 
-func (p *Plugin) filterNodeUsage(node *corev1.Node, nodeMetric *slov1alpha1.NodeMetric, filterProfile *usageThresholdsFilterProfile) *framework.Status {
+func (p *Plugin) filterNodeUsage(node *corev1.Node, nodeMetric *slov1alpha1.NodeMetric, filterProfile *usageThresholdsFilterProfile, estimatedUsed map[corev1.ResourceName]int64) *framework.Status {
```

Comment:
It's recommended here to consider calling the function `estimatedAssignedPodUsed` to get usage of assigned pods, thus the used is equal to totalUsed(from nodeMetrics) +  estimatedUsed(currentPod) + estimatedAssignedPodUsed.

Reply:
ok